packages/node_modules/@ciscospark/widget-meet/src/messages.js

```diff
@@ -22,6 +22,22 @@ export default defineMessages({
     id: `ciscospark.container.meet.button.hangup`,
     defaultMessage: `Hangup`
   },
+  okButtonLabel: {
+    id: `ciscospark.container.meet.button.ok`,
+    defaultMessage: `Ok`
+  },
+  callErrorDeviceNotFound: {
+    id: `ciscospark.container.meet.message.deviceNotFound`,
+    defaultMessage: `No calling devices found`
```

Comment:
Can we change this to: "Cisco Spark cannot find your camera. Please check your browser settings, and that your camera is accessible, and try again."